### PR TITLE
Few wasm fixes

### PR DESCRIPTION
**Pull request by xtuc**
**What kind of change does this PR introduce?**

**Did you add tests for your changes?**

**Does this PR introduce a breaking change?**

**What needs to be documented once your changes are merged?**

### CODE REVIEW

**Comment by xtuc** [comment]
Some tests are failing because of https://github.com/webpack/webpack/pull/7275/files#r189592198

File: package.json [hunk 1]
```diff
@@ -5,9 +5,10 @@
   "description": "Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.",
   "license": "MIT",
   "dependencies": {
-    "@webassemblyjs/ast": "1.5.1",
-    "@webassemblyjs/wasm-edit": "1.5.1",
-    "@webassemblyjs/wasm-parser": "1.5.1",
+    "@webassemblyjs/ast": "1.5.4",
+    "@webassemblyjs/wasm-edit": "1.5.4",
+    "@webassemblyjs/wasm-opt": "1.5.3",
```
Comment (sokra) [inline package.json]: 1.5.4?

Comment (xtuc) [reply package.json]: That's correct, thanks

**Comment by sokra** [comment]
Thanks

**Pull request merged**